mdsuite/transformations/map_molecules.py

```diff
@@ -43,8 +43,11 @@ def __init__(self, experiment: object, molecules: dict):
         experiment : object
                 Experiment object to work within.
         molecules : dict
-                Molecule dictionary to use as reference. e.g. {'emim': {'smiles': 'CCN1C=C[N+](+C1)C', 'amount': 20},
-                                                                'PF6': {'smiles': 'F[P-](F)(F)(F)(F)F', 'amount': 20}}
+                Molecule dictionary to use as reference. e.g.
+                .. code-block:: python
+
+                   {'emim': {'smiles': 'CCN1C=C[N+](+C1)C', 'amount': 20}, 'PF6': {'smiles': 'F[P-](F)(F)(F)(F)F', 'amount': 20}}
+
                 would be the input for the emim-PF6 ionic liquid.
```

Comment:
I just build the docs and `MDSuite/docs/build/html/_transformations/map_molecules.html` is just an empty page

Reply:
If you look at the warnings one of them is that it did not import

Reply:
It compiles for me. What version of sphinx?

Reply:
`Running Sphinx v4.0.1`

Reply:
hmm me too

Reply:
I downloaded the GitHub build documentation, which is all that matters. The map molecules is build there, but the code block does not seem to work

Reply:
I guess it is also missing the blank space above. If that is added it should be solved

Reply:
Please check again.

Reply:
Still not working

Reply:
So for me it does not update at all. I added new lines to the doc string and it has refused to update. I looked into the link and it seems okay.

Reply:
This is how it looks for me

![image](https://user-images.githubusercontent.com/46721498/123289085-6a077a00-d510-11eb-9810-0da7872663a5.png)

But I assume it should be similar to the e.g. the project class one
![image](https://user-images.githubusercontent.com/46721498/123289179-81466780-d510-11eb-869e-cf08be56cac6.png)

Reply:
I fixed it in 955c98c ... I guess we both looked at the wrong place